modules/Microsoft.Compute/images/.test/common/dependencies_rbac.bicep

```diff
@@ -0,0 +1,13 @@
+targetScope = 'subscription'
+
+param managedIdentityResourceId string
```

Comment:
I guess we should add the Metadata here too. Be it only to lead by example

Reply:
GTood catch. Added.